index.html

```diff
@@ -0,0 +1,213 @@
+      rel="stylesheet"
+    />
+    <title>Static Template</title>
+  </head>
+  <body>
+    <div
+      class="three-column-grid__expand-two gap-sm center-wrapper main-layout"
+    >
+      <header class="main-header three-row-grid__expand-two gap-sm">
+        <aside class="Side-bar-menu">
+          <div>
+            <img
+              class="twitter-logo"
+              src="https://img.search.brave.com/jaw-9w6G39GIIxdKdqgSGOIdknwh344YLISmmJE_2rw/rs:fit:1200:1200:1/g:ce/aHR0cHM6Ly9sb2dv/cy1kb3dubG9hZC5j/b20vd3AtY29udGVu/dC91cGxvYWRzLzIw/MTYvMDIvVHdpdHRl/cl9Mb2dvX25ldy5w/bmc"
+              alt="Twitter Logo"
+            />
+          </div>
+          <nav>
+            <ul class="menu-list">
+
+              <li class="active">
+                <img
+                src="/icons/house-black-silhouette-without-door.png"
+                alt="Home-icon"
+                class="active"
+              />
+                <a href="#">
+                  Home
+                </a>
+              </li>
+
+              <li>
+                <img
+                    class="search-icon"
+                    src="/icons/search.png"
+                    alt="search-icon"
+                  />
+                <a class="menu-list-link" href="#">
+                  Explore</a
+                >
+              </li>
+              
+              <li>
+                <img
+                src="/icons/bell.png"
+                alt="notification-icon"
+                />
+                <a class="menu-list-link" href="#">
+                  Notification
+                </a>
+              </li>
+              <li>
+                <img src="/icons/envelope.png" alt="message-icon" /> 
+                <a class="menu-list-link" href="#">
+                  Message
+                </a>
+              </li>
+              <li>
+                <img src="/icons/bookmark-white.png" alt="bookmark-icon" />
+                <a class="menu-list-link" href="#">
+                  Message
+                </a>
+              </li>
+              <li>
+                <img src="/icons/menu.png" alt="list-icon" /> 
+                <a class="menu-list-link" href="#">
+                  Lists
+                </a>
+              </li>
+              <li>
+                <img src="/icons/user.png" alt="profile-icon" /> 
+                <a class="menu-list-link" href="#">
+                  Profile
+                </a>
+              </li>
+              <li>
+                <img src="/icons/more.png" alt="more-icon" /> 
+                <a class="menu-list-link" href="#">
+                  More
+                </a>
+              </li>
+            </ul>
+          </nav>
+          <button class="side-bar-button__tweet">Tweet</button>
+        </aside>
+      </header>
+      <main>
+        <div class="main-box">
+          <h3>Home</h3>
+        </div>
+          <div class="  profile-feed-pic main-box">
```

Comment:
did you mean to open a div "main box" then close it immediately with only a h3 inside? The next div "profile-feed-pic" is indented, which suggests that it wants to be the child of the class above. Should the divs be siblings or parent-child?

Reply:
The code wasn't finalised, the feed icons (e.g the gif ) was imperfect and this snippet was incomplete and was to be used later when i get a hold of the sizing for the icons.